lisa/tools/kill.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Kill(Tool):
```

Comment:
The tool is not necessary so far, the process object can be killed itself.

Reply:
if reuse an existing environment, and already have iperf3 running.

Reply:
1. You can use the `after_case` to kill the iperf3 process. The advantage is, even the test case is failed, there is a chance to kill it.
2. Implement the find by name in the Kill tool also. For example, `def by_name(self, process_name: str) -> None`.

Reply:
how to get nodes in after_case? @squirrelsc

Reply:
It's the same as test case.

Reply:
I passed 'environment: Environment', it doesn't work.

Reply:
update